System.IO.Packaging/FileFormatException.cs

```diff
@@ -61,8 +59,8 @@ public FileFormatException(string message, Exception innerException)
         public FileFormatException(Uri sourceUri)
             : base(
                 sourceUri == null
-                ? SR.FileFormatException
-                : SR.Format(SR.FileFormatExceptionWithFileName, sourceUri))
+                ? Resources.FileFormatException
+                : Formatter.Format(Resources.FileFormatExceptionWithFileName, sourceUri))
```

Comment:
Regardless of my other question regarding `FileFormatException`, this is an example showing how this PR splits `SR` into `Resources` and `Formatter`. This way, the generic formatting methods are separated from the generated properties provided by the `Resources` class. This makes sense because, other than with `Settings`, the `Resources` class is not generated as a partial class.